test/snapshots/prefer-string-starts-ends-with.mjs.md

```diff
@@ -34,6 +46,18 @@ Generated by [AVA](https://avajs.dev).
     `␊
     > 1 | /^a/.test((0, "string"))␊
         | ^^^^^^^^^^^^^^^^^^^^^^^^ Prefer \`String#startsWith()\` over a regex with \`^\`.␊
+    ␊
+    --------------------------------------------------------------------------------␊
+    Suggestion 1/3: For strings that may be \`undefined\` / \`null\`, use string casting.␊
+      1 | String(0, "string").startsWith(('a'))␊
```

Comment:
As you can see here , this one is not correct.

Reply:
Fixed this